templates/config/manifests/bases/clusterserviceversion.yaml.tpl

```diff
@@ -12,12 +12,19 @@ metadata:
     createdAt: {{.CreatedAt}}
     support: {{.Annotations.Support}}
     certified: {{.Annotations.IsCertified}}
+  labels:
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.ppc64le: supported
+    operatorframework.io/arch.arm64: supported
+    operatorframework.io/os.linux: supported
+    operatorframework.io/os.zos: supported
```

Comment:
Not familiar of operatorframework, why are these needed?

Reply:
@a-hilaly I am not sure either, but they asked me to put it on, so I did.
I don't think this means anything. Do you want me to remove it?
https://github.com/aws-controllers-k8s/code-generator/pull/461#pullrequestreview-1594255248

Reply:
I don't think it makes sense, so I will remove it.

Reply:
I asked for this, since if ACK is going to distribute a multi-arch image, in order for ACK to be available on architectures other then amd64, this enables the operator to be listed on those clusters.

Reply:
I see in another comment, that power/s390x aren't supported by EKS, for now we should just have 
```
operatorframework.io/os.linux: supported
operatorframework.io/arch.amd64: supported
operatorframework.io/os.linux: supported
operatorframework.io/arch.arm64: supported
```

Reply:
I have tested it and it works without it, because the container image supports manifest lists!

Reply:
@kahirokunn This has nothing to do with the container. This is needed for distribution of the operator bundles to OLM once released. These annotations instruct the catalog-operator in a cluster what operators can be installed for what architectures. If we leave off these annotations, then the operator will not be able to be installed on any ARM OLM distribution.

Reply:
@acornett21 
I think it is backward compatible since it is not annotated from the beginning.
I have never used OperatorFramework at all, so I don't know how to test it or anything.
Could you please open a PR for OperatorFramework yourself?
Thank you 🙏

Reply:
@kahirokunn Please do not resolve this, this is needed like mentioned. Please include this in this PR.

Reply:
@a-hilaly
I put it in because it was requested, but does it seem to be a problem?

https://access.redhat.com/documentation/ja-jp/openshift_container_platform/4.5/html/operators/olm-enabling-operator-for-multi-arch_osdk-generating-csvs